Modal.js

```diff
@@ -0,0 +1,155 @@
+export class Modal {
+
+    #id = 'modal-version-04/13/2024'
```

Comment:
I generally don't see dates hard coded into code files. That's what [git blame](https://git-scm.com/docs/git-blame) is for.

Reply:
As one might expect, they can make the code look outdated quickly... I want your code to look timeless!

Reply:
I just wanted to hide a time capsule in the id. Fine, I'll change it to something a little more boring. 😢

Reply:
@Geo-Bold to quote The Hitman's Bodyguard, "Boring is always best!"